app/src/main/java/com/example/makeiteven2/adapters/LevelsAdapter.kt

```diff
@@ -63,7 +80,7 @@ class LevelsAdapter(private var mLevelsList : List<Level> = ArrayList(), private
     }
 
     override fun onBindViewHolder(holder: ViewHolder, position: Int) {
-        mLevelsList[position].let { list ->holder.bind(list,mCurrentStage) }
+        mLevelsList[position].let { list ->holder.bind(list,mCurrentStage,mCallBack) }
```

Comment:
why you bind the list? and not the element in the list?
ok got it the parameter name is list should be level